presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloPlugin.java

```diff
@@ -13,22 +13,18 @@
  */
 package com.facebook.presto.accumulo;
 
-import com.facebook.presto.accumulo.udf.AccumuloStringFunctions;
 import com.facebook.presto.spi.Plugin;
 import com.facebook.presto.spi.connector.ConnectorFactory;
 import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableSet;
-
-import java.util.Set;
 
 public class AccumuloPlugin
         implements Plugin
 {
-    @Override
-    public Set<Class<?>> getFunctions()
-    {
-        return ImmutableSet.<Class<?>>builder().add(AccumuloStringFunctions.class).build();
-    }
+//    @Override
+//    public Set<Class<?>> getFunctions()
+//    {
+//        return ImmutableSet.<Class<?>>builder().add(AccumuloStringFunctions.class).build();
+//    }
```

Comment:
delete instead of commenting ?

Reply:
Hi @mayankgarg1990 ,Thanks for the suggestions, changes are made accordingly.